ethstats/ethstats.go

```diff
@@ -485,6 +579,14 @@ func (s *Service) reportLatency(conn *websocket.Conn) error {
 	if err := s.sendStats(conn, actionNodePing, ping); err != nil {
 		return err
 	}
+	// Proxy needs a delegate send here to get ACK
+	if s.backend.ProxiedPeer() != nil {
+		signedMessage := <-sendCh
```

Comment:
More just thinking out loud, now this latency will include the time it takes for the proxy to send a message to the validator and receive from it :(

Reply:
I know, it's hard to remove that, unless we 
- measure the proxy <> validator roundtrip and substract that
- ask serverTime at the validator and only have a one way trip from validator to proxy added